x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/retry.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RetryService } from '@kbn/ftr-common-functional-services';
+
+/*
+ * Retry wrapper for async supertests, with a maximum number of retries
+ */
+export const retry = async <T>({
+  test,
+  retryService,
+  retries,
+  timeout,
+}: {
+  test: () => Promise<T>;
+  retryService: RetryService;
+  retries: number;
+  timeout: number;
+}): Promise<T | Error> => {
```

Comment:
Following up on the comment above, this should be just `Promise<T>`.

Reply:
Thanks